.github/stale.yml

```diff
@@ -0,0 +1,44 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 365 # 1 year
```

Comment:
I wanted to start at 1 year because that should still have a significant impact on the number of stale issues we have right now, and then reduce that time period once we've been using the bot for a while.

Reply:
Makes sense 👍 

Can we add an inline comment to describe that this is a starting value and we should reduce it to `X` down the road? That can help anybody to make the change in the future. WDYT?